opendatahub-io/odh-manifests
# Upgrade superset to v1.3.0

https://issues.redhat.com/browse/ODH-507

To test the PR:
1) Deploy superset
2) Visit the route superset in the namespace opendatahub is deployed in
3) Before logging in click on settings on the top right corner
It should reflect the new version

![image](https://user-images.githubusercontent.com/22432723/137326832-1a93db9c-7a03-45cc-a855-ee479d1622c2.png)
